odl/solvers/iterative/statistical.py

```diff
@@ -121,6 +138,131 @@ def mlem(op, x, data, niter=1, noise='poisson', callback=None):
         raise RuntimeError('unknown noise model')
 
 
+def osmlem(op, x, data, niter=1, noise='poisson', callback=None, **kwargs):
+    """Ordered Subsets Maximum Likelihood Expectation Maximation algorithm.
+
+    Attempts to solve::
+
+        max_x L(data | x)
```

Comment:
I would also point to section 5.3.2 in Mathematical Methods in Image Reconstruction, by F. Natterer and F. Wübelling, SIAM, 2001.